composer.json

```diff
@@ -0,0 +1,60 @@
+{
+    "name": "openeuropa/oe_list_pages",
+    "description": "OpenEuropa List Pages.",
+    "type": "drupal-module",
+    "license": "EUPL-1.2",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "require": {
+        "drupal/core": "^8.7",
+        "drupal/entity_meta_relation": "~1.0-beta5",
+        "drupal/search_api": "~1.17",
+        "php": ">=7.2"
+    },
+    "require-dev": {
+        "composer/installers": "~1.5",
+        "drupal-composer/drupal-scaffold": "~2.2",
+        "drupal/config_devel": "~1.2",
+        "drupal/drupal-extension": "~4.0",
+        "drush/drush": "~9.0@stable",
```

Comment:
"drush/drush": "9.x-dev#93075ab"

Reply:
I suppose it is coming from ewcms (locking version).
Do we have any reason for the locking version of Drush (probably in ewcms exist some specific reason for that)?